models/src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -277,7 +277,7 @@ def _run_mapper(
             use_reentrant=use_reentrant,
         )
 
-    def forward(self, x: Tensor, model_comm_group: Optional[ProcessGroup] = None) -> Tensor:
+    def forward(self, x: Tensor, model_comm_group: Optional[ProcessGroup] = None, **kwargs) -> Tensor:
```

Comment:
Could you add `*` after the x tensor, to ensure that model_comm_group is being called as a kwarg everywhere please?